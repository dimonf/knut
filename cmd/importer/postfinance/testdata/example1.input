Buchungsart:;="Alle Buchungen"
Konto:;="CH4609000000877991229"
Währung:;="CHF"

Buchungsdatum;Avisierungstext;Gutschrift in CHF;Lastschrift in CHF;Valuta;Saldo in CHF

08.03.2022;desc1 ;;-19;08.03.2022;796.44
07.03.2022;desc2;4.95;;07.03.2022;787.44
07.03.2022;desc3;;-1139.6;07.03.2022;

Disclaimer:
Dies ist kein durch PostFinance AG erstelltes Dokument. PostFinance AG ist nicht verantwortlich für den Inhalt.