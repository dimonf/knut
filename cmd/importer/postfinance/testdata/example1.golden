2022-03-07 "desc2"
Expenses:TBD       Assets:Postfinance       4.95 CHF

2022-03-07 "desc3"
Assets:Postfinance Expenses:TBD           1139.6 CHF

2022-03-08 "desc1 bar foo"
Assets:Postfinance Expenses:TBD               19 CHF

